.travis.yml

```diff
@@ -207,7 +207,7 @@ jobs:
         - mkdir build
         - cd build
       script:
-        - cmake -GXcode -DBUILD_TESTING=OFF -DBoost_NO_BOOST_CMAKE=ON -DGLEW_USE_STATIC_LIBS=ON ..
```

Comment:
seems unrelated to topic?

Reply:
oops, seems i didnt edit the history correctly, thought b72fa4aa5 came from master. maybe from o01egs test branch. rebased. good to go.